crmsh/sbd.py

```diff
@@ -174,33 +208,35 @@ def _update_configuration(self):
         utils.sysconfig_set(SYSCONFIG_SBD, **sbd_config_dict)
         bootstrap.csync2_update(SYSCONFIG_SBD)
 
-    def _determine_sbd_watchdog_timeout(self):
+    def _adjust_sbd_watchdog_timeout_with_diskless_and_qdevice(self):
         """
-        When using diskless SBD, determine value of SBD_WATCHDOG_TIMEOUT
+        When using diskless SBD with Qdevice, adjust value of sbd_watchdog_timeout
         """
         if not self.diskless_sbd:
             return
         # add sbd after qdevice started
         if utils.is_qdevice_configured() and utils.service_is_active("corosync-qdevice.service"):
             qdevice_sync_timeout = utils.get_qdevice_sync_timeout()
-            self._sbd_watchdog_timeout = qdevice_sync_timeout + 5
-            if self._is_s390 and self._sbd_watchdog_timeout < 15:
-                self._sbd_watchdog_timeout = 15
+            if self._sbd_watchdog_timeout <= qdevice_sync_timeout:
+                watchdog_timeout_with_qdevice = qdevice_sync_timeout + 5
+                bootstrap.warn("sbd_watchdog_timeout is set to {} for qdevice, it was {}".format(watchdog_timeout_with_qdevice, self._sbd_watchdog_timeout))
+                self._sbd_watchdog_timeout = watchdog_timeout_with_qdevice
+            self._adjust_sbd_watchdog_timeout_for_s390()
```

Comment:
Improvment to simplify code:

1. No need to call _adjust_sbd_watchdog_timeout_for_s390() here. The reason is crmsh hard-coded s390 default is initialized in __init__(), also the above function is called when load profiles.yml in _initialize_sbd(). No other place could have _sbd_watchdog_timeout smaller than the s390 default.

2. Furthermore, this function has duplicate code and can be simplified further

Reply:
OK, remove it from here